intellij-java-google-style.xml

```diff
@@ -1,17 +1,17 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <code_scheme name="GoogleStyle">
     <option name="JAVA_INDENT_OPTIONS">
-        <value>
-            <option name="INDENT_SIZE" value="2" />
-            <option name="CONTINUATION_INDENT_SIZE" value="4" />
-            <option name="TAB_SIZE" value="8" />
-            <option name="USE_TAB_CHARACTER" value="false" />
-            <option name="SMART_TABS" value="false" />
-            <option name="LABEL_INDENT_SIZE" value="0" />
-            <option name="LABEL_INDENT_ABSOLUTE" value="false" />
-            <option name="USE_RELATIVE_INDENTS" value="false" />
-        </value>
-    </option>
+    <value>
```

Comment:
The XML formatting here doesn't look right. Could you go through this and make sure the indentation is correct?

Reply:
Yeah I was lazy and didn't fix that. Will do.

Reply:
These are still incorrectly indented.

Reply:
Each child is one tab in from the parent. How should I do it instead?

Reply:
The <value> tag should be indented on more because the line above it is <option>, not <option/>